Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js

```diff
@@ -352,10 +352,10 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
             );
 
             this._assertLocationNavigationItem(
-                value[0], "Content structure", "content-structure", "/allez/om", "fre-FR-root"
+                value[0], "nav.content.structure", "content-structure", "/allez/om", "fre-FR-root"
```

Comment:
it would be better to write `Y.eZ.Translator.trans("nav.content.structure")` that way in the future we can change the implementation of `Y.eZ.Translator` in the test to validate that the strings are actually translated correctly.